specs/commandline/sign.md

```diff
@@ -27,37 +41,38 @@ Flags:
 
 ## Usage
 
-### Sign a container image
+### Sign an OCI artifact
 
 ```shell
-# Add a key which uses a local private key and certificate, and make it a default signing key
-notation key add --default --name <key_name> <key_path> <cert_path>
-
-# Or change the default signing key to an existing signing key
-notation key update --default <key_name>
+# Prerequisites: 
+# - A signing plugin is installed. See plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
```

Comment:
Instead of the main branch, should we use a tag version for the URL?

Reply:
Can we update it later once we have cut a version for notaryproject?